presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -3220,6 +3220,43 @@ public void testScalarSubquery()
         assertQuery("SELECT * FROM (VALUES 1.0) t(a) WHERE 1=(SELECT count(*) WHERE 1 = a)", "SELECT 1.0");
     }
 
+    @Test
+    public void testMultipleOccurrencesOfCorrelatedSymbol()
```

Comment:
can you please move these tests to `TestSubqueries`. ATQ tests are run per most of the connectors, while tests are focused only on planner.

Reply:
I am aware of that. I will move them as soon as tests with tables can be used in `TestSubqueries`.

Reply:
I was thinking of or replacing `nation` and `region` with `VALUES`.